### ISSUE TITLE
DocCreator/MethodDefinition.java still depends on java8 features

### ISSUE THREAD

**Issue by MoraFermi** [virenerus/tabletoptool#1]
Of course, java8 is going to be the default in near future -- it would, however, be nice to make this project wholly java7 compliant.

**Comment by virenerus** [comment]
Is not that important because the DOcCreator Project is only used to export java comments to the website and is thus not part of the distribution itself.

**Comment by virenerus** [comment]
I removed the project. It is no longer needed.
